packages/core/src/spec/data.d.ts

```diff
@@ -243,6 +244,58 @@ export interface AxisGenomeData {
     channel: PrimaryPositionalChannel;
 }
 
+export interface AxisMeasureData {
+    type: "axisMeasure";
+
+    /**
+     * Which channel's scale domain to monitor.
+     *
+     * __Default value:__ `"x"`
+     */
+    channel?: PrimaryPositionalChannel;
+
+    /**
+     * Optional min measure size in pixels.
+     * It has to be big enough to fit the measure/span label.
+     * The measure size in pixels will vary between this minimum value and
+     * 10 times this value, when it switches to the next smaller span value.
+     *
+     * __Default value:__ `50`
+     */
+    minMeasureSize?: number;
+
+    /**
+     * Optional value of threshold to hide measure.
+     * If measure size in domain units (e.g. DNA bases) is less than or equal
+     * to this threshold, hide the measure to avoid "flickering" caused
+     * by rounding.
+     * The measure is not really necessary at this zoom level,
+     * as the axis ticks are filling that need.
+     *
+     * __Default value:__ `10`
+     */
+    hideMeasureThreshold?: number;
+
+    /**
+     * Optional multiplier value which can be used to create a different set
+     * of measures from the powers of 10: 1, 100, 1000, etc.
+     * e.g. : 5, 500, 5000 for multiplierValue = 5
+     *
+     * __Default value:__ `1`
+     */
+    multiplierValue?: number;
```

Comment:
Hmm. I'm not sure about this. What if someone would like to have 1, 5, 10, 50, 100, 50, etc? No need to implement it now, but how would `multiplierValue` work with that?

I would be careful when adding nice-to-have options, as they may become a nuisance later on.